Source/controls/controller_buttons.cpp

```diff
@@ -260,6 +260,62 @@ string_view ToGenericButtonText(ControllerButton button)
 		return "LS";
 	case devilution::ControllerButton_BUTTON_RIGHTSTICK:
 		return "RS";
+	case devilution::ControllerButton_BUTTON_MENU:
+		return "Menu";
+	case devilution::ControllerButton_BUTTON_VOLUME_UP:
+		return "Volume Up";
+	case devilution::ControllerButton_BUTTON_VOLUME_DOWN:
+		return "Volume Down";
+	case devilution::ControllerButton_BUTTON_DPAD_UP:
+		return "Up";
+	case devilution::ControllerButton_BUTTON_DPAD_DOWN:
+		return "Down";
+	case devilution::ControllerButton_BUTTON_DPAD_LEFT:
+		return "Left";
+	case devilution::ControllerButton_BUTTON_DPAD_RIGHT:
+		return "Right";
+	case devilution::ControllerButton_NONE:
+		return "None";
+	case devilution::ControllerButton_IGNORE:
+		return "Ignored";
+	default:
+		return "Unknown";
+	}
+}
+
+string_view ToGenericButtonText2(ControllerButton button)

```

Comment:
What's this for? RG35xx uses the Nintendo button layout.
Simply `set(DEVILUTIONX_GAMEPAD_TYPE Nintendo)` in `rg35xx_garlic.cmake`.